lib/scripts/create_logs_config.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+LOGS_CONFIG_DIR = os.environ['LOGS_CONFIG_DIR']
+LOGS_CONFIG = os.environ['LOGS_CONFIG']
+
+config = "{\"logs\":" + LOGS_CONFIG + "}" 
+path = LOGS_CONFIG_DIR + "/logs.yaml"
+
+file = open(path, 'w')
+
+print "writing {} to {}".format(config, path)
+file.write(config)
+file.write("\n")
```

Comment:
I'd like to rewrite this a bit. You're not closing the file.

Can you do this instead

```python
with open(path, 'w') as f:
    print "writing {} to {}".format(config, path)
    f.write(config)
    f.write("\n")
```